components/kamstrup_403.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Kamstrup 403",
+  "owner": ["@golles"],
+  "manifest": "https://github.com/golles/ha-kamstrup_403/blob/main/custom_components/kamstrup_403/manifest.json",
```

Comment:
This link goes to a web page, it needs to point to the raw file

Reply:
Updated to raw file, thanks